code/unclassified/src/biggest_of_n_numbers/biggest_of_n_numbers.cs

```diff
@@ -0,0 +1,24 @@
+namespace biggest_of_n_numbers
+{
+    public class biggest_of_n_numbers
+    {
+        public void main()
+        {
+            int n , max , tmp;
```

Comment:
Why are you declaring your variables at the top of the function? Why can't you do something like `int n = Console.ReadLine()` when necessary?